analytics_data_api/v0/serializers.py

```diff
@@ -332,6 +332,14 @@ class LearnerSerializer(serializers.Serializer, DefaultIfNoneMixin):
     enrollment_date = serializers.DateField(source='enrollment_date', format=settings.DATE_FORMAT)
     cohort = serializers.CharField(source='cohort')
 
+    def transform_segments(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
+
+    def transform_cohort(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
```

Comment:
We need to support the concepts of "This user belongs to no cohort" and "This user belongs to the cohort named <the-empty-string>". It used to be that both were represented by the empty string value, but I'm now afraid that both are represented by the `null` value. I think that when a user belongs to no cohort (field is missing in ES), the API should return `null`, and when they belong to a cohort with the name being the-empty-string, we should return the empty string.